common/src/main/resources/data/empoweredpathways/recipes/infused_stonework_block.json

```diff
@@ -2,18 +2,18 @@
   "type": "minecraft:crafting_shaped",
   "pattern": [
```

Comment:
comments here would be ideal to be honest

Reply:
Ok so I added comments above the recipe lmk if that works